src/EventService.Mappers/Db/DbEventUserMapper.cs

```diff
@@ -28,4 +31,25 @@ public class DbEventUserMapper : IDbEventUserMapper
         CreatedAtUtc = DateTime.UtcNow
       }).ToList();
   }
+
+  public List<DbEventUser> Map(
+    CreateEventRequest request,
+    Guid senderId,
+    Guid eventId)
+  {
+    return request is null
+      ? null
+      : request.Users.Select(u => new DbEventUser
+      {
+        Id = Guid.NewGuid(),
+        EventId = eventId,
+        UserId = u.UserId,
+        Status = (request.Access == AccessType.Opened && u.UserId == senderId)
+          ? EventUserStatus.Participant
+          : EventUserStatus.Invited,
```

Comment:
sender=organizer, it can't be in list (you've checked it in validator)
plus one user can only INVITE other (can't make someone participant) => here for all users set status Invited